client/src/components/WebAuthn/Verify.js

```diff
@@ -218,7 +218,6 @@ Verify.propTypes = {
   method: PropTypes.object.isRequired,
   publicKey: publicKeyType,
   onCompleteVerification: PropTypes.func.isRequired,
-  moreOptionsControl: PropTypes.func,
```

Comment:
This was just wrong, if you passed a func here you'd get a "react cannot render functions as children" error. I suspect the old unit test only used to pass because of the shallow() render

Reply:
Functional components are functions and would render fine - so we probably shouldn't just outright remove the prop declaration.

Reply:
Yeah except for the bit where it doesn't actually render :-)

It's used in the render() method
```
      <div className="mfa-verification-container__actions mfa-action-list">
        {failure ? retryButton : null}
        {moreOptionsControl}
      </div>
```

The old unit test did pass a function ` moreOptionsControl={() => 'Hello world'}`

However when I passed a function, I got an error

The new unit test has this instead which works `moreOptionsControl: <div className="test-more-options">Hello world</div>`

Reply:
Right but it is used. So we shouldn't remove the prop definition is my point. 
If the prototype is wrong, it should be updated to whatever is correct.

Reply:
Updated